mars-sim-core/src/main/java/com/mars_sim/core/UnitManager.java

```diff
@@ -43,6 +43,7 @@
 import com.mars_sim.core.time.Temporal;
 import com.mars_sim.core.vehicle.Vehicle;
 import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import com.mars_sim.ui.swing.tool.mission.NavpointPanel;
```

Comment:
@bevans2000 @muybien3433 

I wonder if you guys can figure out together the best way forward for the concept of addDisplayUnit to be applied without importing NavpointPanel class from mars-sim-ui package here.

Reply:
Remove the reference to the UI components from the Unit manager. Get the new getDisplayUnits to combine the Settlement and Vehicle lists which will work but could be slow.
Alternatively, use the Settlement and Vehicles list directly in the layer code. The logic is slightly different on the layer depending upon which Unit type is selected so it would make sense.